src/ui/components/shared/Error.tsx

```diff
@@ -180,7 +217,9 @@ function TrialExpired() {
               expired.
             </div>
           </div>
-          <ActionButton action={"library"} />
+          <ActionButton
+            action={loading || recording?.userRole !== "team-admin" ? "library" : "team-billing"}
```

Comment:
It looks like we should be way past the `loading=true` state if we land on the trial expired modal. I think you're free to take it out here unless you want the redundancy 👍

Reply:
Yeah, I prefer not to assume we'll have queried it so it doesn't bug out later but you're right that it should be safe to ignore.